Java/tests/src/test/java/com/microsoft/recognizers/text/tests/datetime/DateTimeExtractorTest.java

```diff
@@ -124,71 +141,102 @@ private static IDateTimeExtractor getEnglishExtractor(String name) {
 
         IOptionsConfiguration config = new BaseOptionsConfiguration();
         switch (name) {
-            case "DateExtractor":
```

Comment:
These changes were only line break changes? Please be careful to avoid making unnecessary changes unless it was for consistency with the rest of the file. Such changes can lead to less usability in code review tools and more effort required in the reviews.